build_tools/buildkite/scripts/update_pipeline_configurations.py

```diff
@@ -29,13 +29,19 @@
 import requests
 from pybuildkite import buildkite
 
+from common.buildkite_utils import get_pipeline
+
 GIT_REPO = "https://github.com/google/iree"
 PIPELINE_ROOT_PATH = "build_tools/buildkite/pipelines"
 TRUSTED_BOOTSTRAP_PIPELINE_PATH = os.path.join(PIPELINE_ROOT_PATH, "fragment",
                                                "bootstrap-trusted.yml")
 UNTRUSTED_BOOTSTRAP_PIPELINE_PATH = os.path.join(PIPELINE_ROOT_PATH, "fragment",
                                                  "bootstrap-untrusted.yml")
 
+IREE_TEAM_REST_UUID = "1a2cbc72-2c8e-4375-821e-e3dfa1db96b5"
```

Comment:
just checking - this is fine to include, and isn't a secret that should be passed through env vars or some other mechanism?

Reply:
I'm pretty sure it's not a secret and I don't really see any reason why it would be. Ability to edit is guarded behind actual access tokens. Plus I think it's impossible to change, which would be a weird thing if it were a secret token